cmd/kubectl-kubestellar/main.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/kubestellar/kubestellar/cmd/kubectl-kubestellar/cmd"
+)
+
+func main() {
+	flags := pflag.NewFlagSet("kubectl-kubestellar", pflag.ExitOnError)
+	pflag.CommandLine = flags
```

Comment:
What is the point of this replacement?  I see no use of this flag set.  This is not the pattern shown in https://github.com/spf13/pflag#supporting-go-flags-when-using-pflag .  KubectlKubestellarCommand makes and uses a different FlagSet.

Reply:
removed